sdks/cpp/src/agones/sdk.cc

```diff
@@ -43,8 +43,7 @@ bool SDK::Connect() {
 
   // make the health connection
   agones::dev::sdk::Empty response;
-  grpc::ClientContext context;
-  pimpl_->health_ = pimpl_->stub_->Health(&context, &response);
+  pimpl_->health_ = pimpl_->stub_->Health(new grpc::ClientContext(), &response);
```

Comment:
Thanks for fixing this.

If this context needs to live as long as the SDK, I think it should be stored as a SDK data member wrapped in a std::unique_ptr.

Reply:
My C++ is a bit rusty, so while I _think_ I've done what you suggested, please check it closely to make sure I've done it correctly.